TestTools/StreamChatTestTools/Mocks/StreamChat/Workers/CurrentUserUpdater_Mock.swift

```diff
@@ -32,10 +32,11 @@ final class CurrentUserUpdater_Mock: CurrentUserUpdater {
 
     override func updateUserData(
         currentUserId: UserId,
-        name: String? = nil,
-        imageURL: URL? = nil,
-        privacySettings: UserPrivacySettings? = nil,
-        userExtraData: [String: RawJSON]? = nil,
+        name: String?,
+        imageURL: URL?,
+        privacySettings: UserPrivacySettings?,
+        role: UserRole?,
+        userExtraData: [String: RawJSON]?,
```

Comment:
Removed default values for since that is why we forgot to add privacySettings to CurrentUser's method.